conn.go

```diff
@@ -196,35 +206,41 @@ func (c *conn) serveWrite() {
 	}
 }
 
+//todo maybe refactor this
 func (c *conn) serveRead() {
 	defer c.Close()
+
 	var event string
+
 	for {
 		var header parser.Header
+
 		if err := c.decoder.DecodeHeader(&header, &event); err != nil {
-			c.onError("", err)
+			c.onError(rootNamespace, err)
 			return
 		}
-		if header.Namespace == "/" {
-			header.Namespace = ""
+
+		if header.Namespace == aliasRootNamespace {
+			header.Namespace = rootNamespace
 		}
+
 		switch header.Type {
 		case parser.Ack:
 			conn, ok := c.namespaces[header.Namespace]
 			if !ok {
-				c.decoder.DiscardLast()
+				_ = c.decoder.DiscardLast()
```

Comment:
Not sure about this `_ = ` is it a go best practice or just personal style?

Reply:
I wrote at https://github.com/googollee/go-socket.io/pull/390#discussion_r489957901 why i am used underscore